doc/ledger.txt

```diff
@@ -78,13 +80,23 @@ Tips and useful commands
   value is 1). A separator between the commodity and the amount may be set
   using `g:ledger_commodity_sep`.
 
-  It is convenient to define a mapping like the following:
+  See below for the recommended mappings.
+
+* `:call ledger#autocomplete_and_align()`
+
+  when the cursor is on a number or immediately after it, invokes
+  `ledger#align_amount_at_cursor()` to align it and add the default currency;
+  otherwise, performs autocompletion. If you define the following mappings
+  then you may perform both autocompletion and alignment using the <Tab> key:
 
-        au FileType ledger inoremap <silent><buffer> <C-l> \
-        <Esc>:call ledger#align_amount_at_cursor()<CR>
+        au FileType ledger inoremap <silent> <Tab> \
+        <C-r>=ledger#autocomplete_and_align()<CR>
+        au FileType ledger vnoremap <silent> <Tab> :LedgerAlign<CR>
```

Comment:
Those definitions are meant to be put in `.vimrc` and those mappings are defined only for buffers with a `ledger` filetype, so they seem ok to me. Alternatively, we may suggest to create `after/ftplugin/ledger.vim` and use `[iv]noremap <buffer>` there.